apps/festival/festival/src/app/marketplace/title/view/view.component.scss

```
@@ -11,5 +11,47 @@ movie-header-cta {
   a, button {
     width: fit-content;
   }
+
+  event-request-screening {
+    display: flex;
+    border-radius: 26px;
+    animation: pulse 2s infinite cubic-bezier(0.20, 0, 0, 0.5);
+
+    @keyframes pulse { 
+      from {
+        box-shadow: 0 0 0 0 rgba(87, 95, 148, 0.64);
+      }
+      to {
+        box-shadow: 0 0 0 15px rgba(232, 76, 61, 0);
+      }
+    }
+  }
+
+  .linear-wipe {
+    text-align: center;
+    
+    background: linear-gradient(to right, #FFF 75%, #849fff 90%, #FFF 95%);
```

use full colors instead of shortcut (stay coherent) : 
```suggestion
    background: linear-gradient(to right, #ffffff 75%, #849fff 90%, #ffffff 95%);
```